kasa/smart/smartchilddevice.py

```diff
@@ -38,7 +39,7 @@ async def update(self, update_children: bool = True):
     @classmethod
     async def create(cls, parent: SmartDevice, child_info, child_components):
```

Comment:
```suggestion
    async def create(cls, parent: SmartDevice, child_info, child_components) -> Self:
```
this would also work now, I think.